src/components/Blog/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import { Row, Col, Container, Image } from "react-bootstrap";
+import "./index.css";
+
+const Blog = (props) => {
+  const storyID = props.match.params.id;
+  const _storyURL = `https://public-api.wordpress.com/wp/v2/sites/worldwithwomen30560421.wordpress.com/posts/${storyID}`;
+  console.log(storyID);
```

Comment:
We shouldn't leave console.logs when deploying

Reply:
removed